tests/unit/test_config.py

```diff
@@ -157,10 +157,10 @@ def test_get_default_environ(self):
         if "GLOBUS_SDK_ENVIRONMENT" in os.environ:
             prev_setting = os.environ["GLOBUS_SDK_ENVIRONMENT"]
             del os.environ["GLOBUS_SDK_ENVIRONMENT"]
-        self.assertEqual(globus_sdk.config.get_default_environ(), "default")
+        self.assertEqual(globus_sdk.config.get_globus_environ(), "default")
         # otherwise environ value
         os.environ["GLOBUS_SDK_ENVIRONMENT"] = "beta"
-        self.assertEqual(globus_sdk.config.get_default_environ(), "beta")
+        self.assertEqual(globus_sdk.config.get_globus_environ(), "beta")
 
```

Comment:
Wouldn't hurt to add a test for the changed behavior here

`os.environ["GLOBUS_SDK_ENVIRONMENT"] = "production"`
`self.assertEqual(globus_sdk.config.get_globus_environ(), "default")`